### PR TITLE
Add custom user agent

### PR DESCRIPTION
**Pull request by aevv**

### CODE REVIEW

File: src/codat.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ class Api {
       qs: args,
       json: true,
       headers: {
-        Authorization: `Basic ${btoa(this.__apiKey)}`
+        Authorization: `Basic ${btoa(this.__apiKey)}`,
+        'User-Agent': 'Codat-node-client/2.6.0'
```
Comment (jasond-s) [inline src/codat.js:41]: the package file is just JSON, is it seen as bad form to `require` it and use a property ref?

**Review by jasond-s — APPROVED** [review]
Seems all good, just a small maintainability point.

File: src/codat.js (hunk 1 above)
Comment (aevv) [reply src/codat.js:41]: i have no idea

Comment (aevv) [reply src/codat.js:41]: https://stackoverflow.com/questions/48609931/how-can-i-reference-package-version-in-npm-script has a few ways but are all a bit jank

Comment (aevv) [reply src/codat.js:41]: i dont think referencing package.json is the answer as thats at compile time, but this code will be running after download in a different context - we might pull the version of a different package.json

**Pull request merged**